<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Dharsanchlm" failures="1" tests="1" name="Test" time="8.331" errors="0" timestamp="2025-09-10T19:03:15 IST">
  <testcase classname="com.runner.NaukriRunner" name="NakuriLogin" time="0.408">
    <failure type="java.lang.AssertionError" message="Error:Occur During the value passing">
      <![CDATA[java.lang.AssertionError: Error:Occur During the value passing
at org.junit.Assert.fail(Assert.java:89)
at com.NakuriProject.BaseClass.inputPassing(BaseClass.java:75)
at com.pageobjectmodel.NakuriLoginPage.validLogin(NakuriLoginPage.java:33)
at com.runner.NaukriRunner.NakuriLogin(NaukriRunner.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- NakuriLogin -->
</testsuite> <!-- Test -->
